app/pages/sections/catalogue-solutions/order-item/contextCreator.test.js

```diff
@@ -130,4 +138,77 @@ describe('catalogue-solutions order-item contextCreator', () => {
       expect(context.saveButtonText).toEqual(manifest.saveButtonText);
     });
   });
+  describe('getErrorContext', () => {
+    const mockValidationErrors = [{
+      field: 'quantity',
+      id: 'quantityRequired',
+    },
+    {
+      field: 'price',
+      id: 'priceRequired',
+    }];
+    const manifestWithErrors = {
+      questions:
+      [{
+        id: 'quantity',
+        mainAdvice: 'Quantity',
+        rows: 3,
+        error: {
+          message: 'quantity error',
+        },
+      }],
+      addPriceTable:
+      {
+        data: [[{
+          question: {
+            type: 'input',
+            id: 'price',
+          },
+        }]],
+      },
+      errorMessages:
+      {
+        quantityRequired: 'Enter a quantity',
+        priceRequired: 'Enter a price',
+      },
+    };
+
+    afterEach(() => {
+      errorContext.getSectionErrorContext.mockReset();
+    });
+
+    it('should call getSectionErrorContext with correct params', () => {
+      errorContext.getSectionErrorContext
+        .mockReturnValue(manifestWithErrors);
+
+      const params = {
+        orderId: 'order-id',
+        solutionName: 'solution-name',
+        serviceRecipientName: 'recipient-name',
+        odsCode: 'ods-code',
+        selectedPrice,
+        validationErrors: mockValidationErrors,
+      };
+
+      getErrorContext(params);
+      expect(errorContext.getSectionErrorContext.mock.calls.length).toEqual(1);
+    });
+
+    it('should call add error message to the table data', async () => {
```

Comment:
think this should be `'should add error message to the table data'` if I'm understanding it correctly